tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1723,13 +1723,13 @@ macro_rules! subscriber_impl_body {
         }
 
         #[inline]
-        fn max_level_hint(&self) -> Option<LevelFilter> {
+        fn max_level_hint(&self) -> ::core::option::Option<LevelFilter> {
```

Comment:
this macro is only used internally; it's not publicly re-exported. i think it would be fine to continue using the prelude, or add a module-level import here?